### PR TITLE
Tag outdial session

### PR DESCRIPTION
**Pull request by avoylenko**
Adding support to attach customer tags for outdial call session. Before it was not possible to do so, this gives more flexibility to maintain the call flow.

Jambonz verbs must be updated as well. Awaiting your comments/suggestions. 

https://github.com/jambonz/verb-specifications/pull/38

### CODE REVIEW

**Comment by avoylenko** [comment]
@davehorton @xquanluu is there anything I can provide you with to speed up the review?

File: lib/tasks/dial.js [hunk 1]
```diff
@@ -100,6 +100,7 @@ class TaskDial extends Task {
     this.referHook = this.data.referHook;
     this.dtmfHook = this.data.dtmfHook;
     this.proxy = this.data.proxy;
+    this.tag = this.data.tag;
```
[the remark this answers is not in the document]
Comment (avoylenko) [reply lib/tasks/dial.js:103]: sure https://github.com/jambonz/verb-specifications/pull/38

[the remark this answers is not in the document]
Comment (davehorton) [reply lib/tasks/dial.js:103]: please update PR to verb-specifications@v0.0.53

[the remark this answers is not in the document]
Comment (davehorton) [reply lib/tasks/dial.js:103]: and rebase

[the remark this answers is not in the document]
Comment (avoylenko) [reply lib/tasks/dial.js:103]: It seems i failed with rebase, so I opened a new PR https://github.com/jambonz/jambonz-feature-server/pull/643

**Pull request closed without merging**